submissions/EnF1NeeT/DOM-task/script.js

```diff
@@ -0,0 +1,169 @@
+<<<<<<< HEAD
+const postsList  = [
+||||||| 76728da
+const postsLst = [
+=======
+const postsList = [
+>>>>>>> 2707486fe7b80d419dcc0d7f8c3d8c626d02b823
```

Comment:
After changing you don't delete pull messages

Reply:
Done